module-4/backend/week_four.md

```diff
@@ -4,8 +4,56 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-TBD  
+* What's the purpose of a project management tool?
+
+To track user stories, bugs, and other tasks. Then assign these tasks to different members of the team and to sprints.
+
+* What are the stages of a card when using Pivotal Tracker?
+
+Not started, started, finished, delivered, accepted, rejected
+
+* What are the four types of cards available in Pivotal Tracker?
+
+Feature, bug, chore, release
+
+* What's your favorite project management tool?
+
+I've worked with Waffle, Trello, PivotalTracker, and Jira. So far, I like PT the best, but I found the sprint management more intuitive in Jira.
+
+* Why do we create staging environments?
+
+So we can test our application before delivery to the client, especially to test any issues with deployment.
+
+* What are the characteristics of a good README (in your opinion)?
+
+Try to say what you need to say in as few words as possible (less is more!). Illustrate your explanations with lots of examples. Have a FAQ section so you can highlight important issues with installation, dependencies, etc.
+
+* What's one main improvement you're going to make to your code regarding accessibility issues?
+
+Make sure the color scheme is colorblind friendly.
+
+* What are some basic security concerns to be aware of when building applications?
+
+I'm not sure. I missed that lesson.
+
+* Why is continuous integration helpful/important?
+
+I'm not sure. I missed that lesson.
+
+* What are some continuous integration tools?
+
+I've heard of Travis CI.
 
 #### Review  
 
-TBD
+* What are some characteristics of "good" git workflow?
+
+Frequent commits, commit headlines less than 80 characters. When creating a PR include full summary of what the PR is doing, including notes for dependencies, screenshots, and supplemental material if necessary. Facilitate discussion on the PR with comments.
+
+* What are the four fundamental concepts of object oriented programming?
+
+I'm not really sure? I guess one of them is to create objects and attach methods to those objects rather than passing objects into methods?
```

Comment:
encapsulation, inheritance, polymorphism, and abstraction. i highly suggest reviewing these four concepts 👍